R/PipeOpClassifAvg.R

```diff
@@ -71,20 +71,12 @@
 PipeOpClassifAvg = R6Class("PipeOpClassifAvg",
   inherit = PipeOpEnsemble,
   public = list(
-    initialize = function(innum = 0, id = "classifavg", param_vals = list()) {
-      super$initialize(innum, id, param_vals = param_vals, prediction_type = "PredictionClassif", packages = "stats")
+    initialize = function(innum = 0, collect = FALSE, id = "classifavg", param_vals = list()) {
```

Comment:
is/should there be any difference in the return value when used with `collect = FALSE`? Just out of curiosity. I guess not, so this simply allows to work for `Multiplicities`?

Reply:
The "collect" only makes the PO "multiplicity-aware", so that it treats multiplicities explicitly. It does not change the return object (other than the fact that multiplicity-oblivious POs return multiplicities when given multiplicities etc., but that is a boring detail)